src/main/java/duke/model/sale/SaleList.java

```diff
@@ -0,0 +1,88 @@
+package duke.model.sale;
```

Comment:
This class is unnecessary since it is merely a copy of UniqueEntityList. Inherit UniqueEntityList instead